src/styles.js

```diff
@@ -56,7 +56,7 @@ export const GlobalStyle = createGlobalStyle`
 
 	body {
 		margin: 0;
-		padding: 50px 100px 50px 100px;
+		padding: 10px 10px 10px 10px;
```

Comment:
Here you can use the shorthand:
```suggestion
		padding: 10px;
```